### DISCUSSION TITLE
Restaurar Metadatos de Monedas de Color y otras Carteras Inteligentes desde Respaldo

### DISCUSSION THREAD

**Discussion by ChemiVilches** [Chia-Network/chia-blockchain#9767] [Support]
Después de actualizar y reiniciar la versión 1.2.11 no guarda las calves. He tenido que utilizar las palabras claves. Ahora me pide "Restaurar Metadatos de Monedas de Color y otras Carteras Inteligentes desde Respaldo", pero no sé qué archivo necesito. 
Si pruebo a saltar este paso no inicia la aplicación. Por favor, ayuda.

![screenshot_chia](https://user-images.githubusercontent.com/97389273/148681717-dd3e0eef-1179-4fc3-a941-9a3a1132079f.png)

**Comment by ldanieltc** [comment]
hola, pudiste resolverlo? me pasa lo mismo y no puedo iniciar.
